sw/device/lib/testing/entropy_testutils.c

```diff
@@ -61,73 +72,30 @@ void entropy_testutils_auto_mode_init(void) {
           {
               .len = 1, .data = {0x00001003},  // One generate returns 1 block.
           },
-      .reseed_interval = 1,  // Reseed after every generate.
+      .reseed_interval = 4,  // Reseed after every 4 generates.
   };
   CHECK_DIF_OK(dif_edn_set_auto_mode(&edn1, edn1_params));
 }
 
-static void setup_entropy_src(void) {
-  dif_entropy_src_t entropy_src;
-  CHECK_DIF_OK(dif_entropy_src_init(
-      mmio_region_from_addr(TOP_EARLGREY_ENTROPY_SRC_BASE_ADDR), &entropy_src));
-
-  // Disable entropy for test purpose, as it has been turned on by ROM
-  CHECK_DIF_OK(dif_entropy_src_set_enabled(&entropy_src, kDifToggleDisabled));
-
-  const dif_entropy_src_config_t config = {
-      .fips_enable = true,
-      .route_to_firmware = false,
-      .single_bit_mode = kDifEntropySrcSingleBitModeDisabled,
-  };
-  CHECK_DIF_OK(
-      dif_entropy_src_configure(&entropy_src, config, kDifToggleEnabled));
-}
+void entropy_testutils_boot_mode_init(void) {
```

Comment:
any suggestions for a better name? this really should be....entropy_src_fips_edn_boot... it's kind of a handful. Or do we really want this to just become auto_mode? If yes, i can do that downstream changes and make this truly boot_mode.

Reply:
I think either option is ok. `entropy_testutils_auto_mode_init()` may be easier to maintain as most test cases calling this function are probably expecting the auto mode behavior.